src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -18224,6 +18272,11 @@ def stack(
         sort : bool, default True
             Whether to sort the levels of the resulting MultiIndex.
         """
+        # stack() may create a column that includes values from multiple input
+        # columns. Tracking types in that case is not simple, so we don't
+        # handle the client-side timedelta type as an input.
+        self._raise_not_implemented_error_for_timedelta()
```

Comment:
Is there a ticket to track stack and unstack for timedelta as well?

Reply:
No, there are a few methods that don't work with timedelta, and we don't plan to implement them. We'll file tickets for them as needed in response to user feedback.